src/components/Navigation.js

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+import '../index.css';
+
+export default function Navigation() {
+  return (
+    <div>
+      <h1>Math Magicians</h1>
+      <nav
+        style={{
+          borderBottom: 'solid 1px',
+          paddingBottom: '1rem',
+        }}
+      >
+        <Link to="/">Home</Link>
+        {' '}
+        |
+        {' '}
+        <Link to="/calculator">Calculator</Link>
+        {' '}
+        |
+        {' '}
+        <Link to="/quote">Quote</Link>
+      </nav>
+    </div>
```

Comment:
- [ ] Kindly ensure your layout matches the design given currently it doesn't below is a comparison  for more detailed look on the differences

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/177616328-e88c9faf-8ae6-45b7-b4d7-6204bfdb8465.png) |  ![image](https://user-images.githubusercontent.com/61408860/177616384-61edec43-96c2-48f0-bc43-a3e42b01cc0a.png)

Reply:
I think this module is not for designing. its for Learning React. Any how on your request I have changed the layout. Here is a preview.
![image](https://user-images.githubusercontent.com/13853450/177636182-65aa681b-e29e-4eb7-ab43-a0c6f48cca0f.png)